harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -210,6 +348,95 @@ def _set_data_loaders(self) -> None:
                     )
                 self.validation_loader = validation_data
 
+    def _step_batch(self):
+        self.batches_trained += 1
+
+        # True epoch-based training is not supported. Epoch start/end is calculated with batch.
```

Comment:
what did we decide about support true epoch-based training?

Like, what's the order we plan on landing things, I mean.  I think we talked about it but I forgot.